compiler/pipes/code-gen.h

```diff
@@ -14,15 +14,14 @@
 class CodeGenerator;
 
 class CodeGenF final : public SyncPipeF<FunctionPtr, WriterData> {
-  using need_profiler = std::false_type;
+  using need_profiler = std::true_type;
```

Comment:
Why was it disabled before?

Reply:
Because it had no logic for each function, it had only `on_finish`. That's why, it was disabled, as it always showed zero measurements.
Now, I've implemented `CodeGenF::execute` that works in parallel.